website/docs/d/pi_cloud_connection.html.markdown

```diff
@@ -20,15 +18,12 @@ data "ibm_pi_cloud_connection" "example" {
 ```
 
 **Notes**
-
 - Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
 - If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
-
   - `region` - `lon`
   - `zone` - `lon04`
 
-  Example usage:
-
+Example usage:
```

Comment:
This example is part of the second bullet point—no need to remove a tab.

Reply:
It's the same problem that was in the list-catalog refactor. This is a common problem for 90% of the existing markdown files.

The change changes it from:
![Screenshot 2024-01-24 at 9 08 16 AM](https://github.com/IBM-Cloud/terraform-provider-ibm/assets/90468712/189cb745-b3fd-4441-b011-b9fc01036245)

to:
![Screenshot 2024-01-24 at 9 09 01 AM](https://github.com/IBM-Cloud/terraform-provider-ibm/assets/90468712/ce982dd9-7bc9-4214-ace1-cc791b294a99)